sw/vendor/README.md

```diff
@@ -1,18 +1,9 @@
-# Work with software code in external repositories
-
-Please refer to [vendor_hw](../../doc/vendor_hw.md) for more details on
-rationale of vendoring external repositories.
+# Third-Party/External Code
 
+This directory contains "vendored" code, i.e. code which copied into this repository from external sources.
+Directory names generally follow the scheme `<vendor>`_`<library>`.
 
-## RISC-V Compliance Suite
+## Work with software code in external repositories
 
-The RISC-V Compliance Suite defines a series of tests that to ensure the
-RISC-V system is complies with the RISC-V specification.  Normally such a test
-would be run at the core level instead of the system level, but doing so confers
-advantages to ensure system integration did not alter the original properties
-of the core (for example breaking byte or half-word accesses).
-
-To vendor in RISC-V Compliance Suite, please use the commands below.
-```
-./util/vendor_hw.py sw/vendor/riscv_compliance.vendor.hjson --verbose
-```
+Please refer to [vendor_hw](../../doc/ug/vendor_hw.md) for more details on
```

Comment:
Can we link here an absolute URL to the docs.opentitan.org site?

Reply:
The reason why I left it like this is that 1) it will not be rendered on Hugo, so having this work with CLI and github seemed sensible and 2) the sw/README.md uses the same format so I assumed it makes sense :)
Just did a quick check, only a handful occurrences of this linking format, should we just get rid of them all?

Reply:
Since it is not rendered, I thought having a link like `https://docs.opentitan.org/doc/ug/vendor_hw/` would help users to get a comfortable version of the content while looking at this file in a text editor.

But I don't think there's a right or wrong here, whatever you feel is most appropriate is OK with me.

Reply:
That makes sense! Added the link.